test/tools/test_save_notes.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+import time
+from unittest.mock import MagicMock
+
+import pytest
+
+CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(f"{CURRENT_DIR}/../../tools")
+sys.path.append(f"{CURRENT_DIR}/../../lib")
+
+# pylint: disable=import-error,wrong-import-position
+from save_notes import save_notes
+from db import DB
+
+
+def test_invalid_string_timestamp():
+    with pytest.raises(ValueError):
+        save_notes(
+            "test_project_id",
+            [
+                {
+                    "note": "This is my note",
+                    "timestamp": 'string_instead_of_time',
+                    "detail_name": "Test_Container",
+                }
+            ],
+        )
+
+def test_string_timestamp():
+    mock_db = DB()
+    mock_db.query = MagicMock()
+
+    save_notes(
+        "test_project_id",
+        [
+            {
+                "note": "This is my note",
+                "timestamp": str(int(time.time_ns() / 1000)),
+                "detail_name": "Test_Container",
+            }
+        ],
+    )
+    mock_db.query.assert_called_once()
```

Comment:
I actually wouldn't mock this, I'd make the DB call and make sure the note is inserted properly. Our test suite is small enough that the time savings on mocking here is minimal, and this way we make sure the DB call is correct in save_notes